modules/end-user-guide/examples/proc_using-the-che-prod-editor-file-to-select-an-ide.adoc

```diff
@@ -0,0 +1,17 @@
+[id="using-the-che-prod-editor-file-to-select-an-ide_{context}"]
+== Using the {prod-short} editor file to select an IDE
+
+Use the `che-editor.yaml` file to define a default IDE for the project users. For a list of supported IDs, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[Optional parameters for the URLs for starting a new workspace].
+
+.Procedure
+
+* Place the `che-editor.yaml` file in the `.che` folder in the root of your project, and point it at the ID of the selected IDE:
++
```

Comment:
```suggestion
+
.Example
```

Reply:
`.Procedure` is correct. See https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs

Reply:
;.Procedure; is correct, but so is introducing an example using the `.Example` heading. Whaty is not correct is the Markdown syntax of the block delimiters :)